### PR TITLE
Add note to reminder user to add quality pattern in PP renaming

### PR DESCRIPTION
**Pull request by fernandog**
@TagForce until we change DB to fix this

UPDATE: the name patterns that SR have, only one have "quality" in it. Won't this cause issues if user select them?

### CODE REVIEW

**Comment by TagForce** [comment]
@fernandog Yeah, well... Quality is a bit of a strange entity in SR at the moment. It gets the quality from the release_name, and it stores it in the db as a bitwise addition to the status int. It's a bit weird. I'll have to check to see if it uses the release_name afterwards, though.

**Comment by fernandog** [comment]
@TagForce about the PR itself, do you agree?

**Comment by TagForce** [comment]
@fernandog Not sure... It does not magically lose the quality info. It gets lost sometimes but I'm not convinced it's because there's no quality in the filename. It should use the release_name for that as that is the original name... It's 'under investigation' as I rewrite the quality tracking, along with implementing a db change and mediainfo scanning. So it'll be a while before it's done.

Ah, stuff it. Go ahead and merge this. It's just text. Won't hurt and is easily changed back later.

**Comment by fernandog** [comment]
@TagForce @abeloin should we revert that?

**Comment by TagForce** [comment]
No, not just yet.

**Comment by abeloin** [comment]
@fernandog I would have said yes but ...
@TagForce Something special planned? New feature??

**Comment by TagForce** [comment]
@abeloin Not really a feature... It still loses quality info sometimes. It can be reverted if you want, though. It shouldn't happen, but apparently does.

I'm working on re-thinking the whole quality setting in SR... Should make it more reliable, and also gives the user more control over what is wanted, and what is not. For instance, it seperates quality from status. It will also change the way it gets the quality (and the filename after post processing is not one of them), and will allow you to use mediainfo.

In short... when all of it is finished, there will be no need to use the quality info in the filename. But it will be some time before it is.
